widgets/hero/styles/default.less

```diff
@@ -112,6 +113,14 @@
 		}
 	}
 
+	& when ( @nav_always_show_mobile = true ) {
+		@media (max-width: @responsive_breakpoint) {
+			& .sow-slider-pagination, & .sow-slide-nav {
+				display: block !important;
```

Comment:
This should rather be implemented in the JS where the showing/hiding of navigation is currently being handled, instead of adding CSS with `!important` which is hard to change.